### PR TITLE
New: Nested Tabs - Style the box section [ED-7335]

### PR DESCRIPTION
**Pull request by hein-obox**

### CODE REVIEW

File: modules/tabs-v2/assets/scss/frontend.scss [hunk 1]
```diff
@@ -17,12 +17,14 @@
 	--tabs-v2-tabs-wrapper-width: initial;
 
 	--tabs-v2-height: initial;
-	--tabs-v2-overflow-x: hidden;
-	--tabs-v2-overflow-y: auto;
+	--tabs-v2-overflow-x: initial;
+	--tabs-v2-overflow-y: initial;
```
Comment (udidol) [inline modules/tabs-v2/assets/scss/frontend.scss:21]: Why did you need to change this?

File: modules/tabs-v2/assets/scss/frontend.scss [hunk 2]
```diff
@@ -134,8 +137,7 @@
 	.elementor-tab-title,
 	.elementor-tab-title:before,
 	.elementor-tab-title:after,
-	.e-container,
-	.elementor-tabs-content-wrapper {
```
Comment (udidol) [inline modules/tabs-v2/assets/scss/frontend.scss:138]: Why delete this selector?

File: modules/tabs-v2/widgets/tabs-v2.php [hunk 3]
```diff
@@ -327,6 +330,13 @@ protected function register_controls() {
 				'fit' => esc_html__( 'Fit to content', 'elementor' ),
 				'height' => esc_html__( 'Height', 'elementor' ),
 			],
+			'selectors_dictionary' => [
+				'fit' => '--tabs-v2-overflow-x: initial; --tabs-v2-overflow-y: initial;',
```
Comment (udidol) [inline modules/tabs-v2/widgets/tabs-v2.php:334]: If you have this as a default value, why do you need to set this as default in the CSS as well?

File: modules/tabs-v2/widgets/tabs-v2.php [hunk 4]
```diff
@@ -523,6 +533,66 @@ protected function register_controls() {
...
+			Group_Control_Background::get_type(),
+			[
+				'name' => 'box_background_color',
+				'types' => [ 'classic', 'gradient' ],
+				'exclude' => [ 'image' ],
+				'selector' => '{{WRAPPER}} .elementor-tabs-content-wrapper',
+			]
+		);
+
+		$this->add_group_control(
```
Comment (udidol) [inline modules/tabs-v2/widgets/tabs-v2.php:552]: Make sure the labels for Border width and border color are same as in the spec (You can use `'fields_options'` for that)

File: modules/tabs-v2/widgets/tabs-v2.php [hunk 5]
```diff
@@ -523,6 +533,66 @@ protected function register_controls() {
...
+			Group_Control_Border::get_type(),
+			[
+				'name' => 'box_border',
+				'selector' => '{{WRAPPER}} .elementor-tabs-content-wrapper',
+			]
+		);
+
+		$this->add_responsive_control(
```
Comment (udidol) [inline modules/tabs-v2/widgets/tabs-v2.php:560]: I tried to test the border radius control and it didn't work for me.
Can you double check it?

File: modules/tabs-v2/widgets/tabs-v2.php [hunk 6]
```diff
@@ -523,6 +533,66 @@ protected function register_controls() {
...
+				'name' => 'box_shadow_box_shadow',
+				'selector' => '{{WRAPPER}} .elementor-tabs-content-wrapper',
+				'condition' => [
+					'box_height!' => 'height',
+				],
+			]
+		);
+
+		$this->add_responsive_control(
+			'box_padding',
```
Comment (udidol) [inline modules/tabs-v2/widgets/tabs-v2.php:584]: I tried to test the padding control and it didn't work for me.
Can you double check it?

**Review by udidol — CHANGES_REQUESTED** [review]

File: modules/tabs-v2/assets/scss/frontend.scss (hunk 1 above)
Comment (hein-obox) [reply modules/tabs-v2/assets/scss/frontend.scss:21]: @udidol 

overflow was causing issue with the box shadow > therefore I removed it by default

File: modules/tabs-v2/widgets/tabs-v2.php (hunk 3 above)
Comment (hein-obox) [reply modules/tabs-v2/widgets/tabs-v2.php:334]: css variable removed

File: modules/tabs-v2/widgets/tabs-v2.php (hunk 4 above)
Comment (hein-obox) [reply modules/tabs-v2/widgets/tabs-v2.php:552]: Updated

File: modules/tabs-v2/widgets/tabs-v2.php (hunk 6 above)
Comment (hein-obox) [reply modules/tabs-v2/widgets/tabs-v2.php:584]: See above

File: modules/tabs-v2/widgets/tabs-v2.php (hunk 5 above)
Comment (hein-obox) [reply modules/tabs-v2/widgets/tabs-v2.php:560]: Test: https://drive.google.com/file/d/1taXlHw33fbol_0HQrVeZhi_9cIpItTVa/view

It seems to work fine on my side.

File: modules/tabs-v2/assets/scss/frontend.scss (hunk 2 above)
Comment (udidol) [reply modules/tabs-v2/assets/scss/frontend.scss]: That's a decision for product to make. Did you approve it with Aviv?

File: modules/tabs-v2/widgets/tabs-v2.php (hunk 5 above)
Comment (udidol) [reply modules/tabs-v2/widgets/tabs-v2.php:641]: Re-tested, works.

File: modules/tabs-v2/widgets/tabs-v2.php (hunk 6 above)
Comment (udidol) [reply modules/tabs-v2/widgets/tabs-v2.php:665]: Re-tested, works.

File: modules/tabs-v2/assets/scss/frontend.scss (hunk 2 above)
Comment (hein-obox) [reply modules/tabs-v2/assets/scss/frontend.scss]: @avivu 
By default the border width are 2pxs, I have changed that to 1px. Is that correct? That how it was in the existing Tabs widget.

**Review by udidol — APPROVED** [review]
